README.md

```diff
@@ -78,10 +78,88 @@ This project is built with specific integrations to enhance its functionality an
 
 ### Decap CMS Integration
 
-We utilize Decap CMS for content management, allowing non-technical team members to update website content easily. The CMS is configured to interact seamlessly with our project's file structure.
+We use [Decap](https://decapcms.org/docs/intro/), formerly known as Netlify CMS, for content management, allowing non-technical team members to update website content easily.
 
-- **CMS Configuration**: The CMS configuration file is located at public/admin/config.yml. This file contains all the settings related to the CMS, including content types, fields, and authentication settings. Please refer to this file for adjusting the CMS interface and functionality according to the project's needs.
-- **Content Directory**: All the content managed through Decap CMS is stored in src/app/content/. This directory includes various markdown files that the CMS edits. Each file represents a different section or page of the website, structured for easy editing and updates
+Decap is a git-based content management system, meaning that content is managed just like any file in our repository, using git for version control.
+
+All the content managed through Decap CMS is stored in `src/app/content/.` This directory includes various markdown files that the CMS edits. Each file represents a different section or page of the website, structured for easy editing and updates
+
+#### Decap Configuration
+
+Decap's setup includes two configuration files:
+
+1. `public/admin/config.yml` - Primarily contains the schema and metadata for the content, but also includes authentication settings among other things,
+2. `public/admin/index.html` - The HTML template to view and edit the content in a web interface. Access it via [/admin/index.html](http://localhost:3000/admin/index.html#/)
+
+Additionally, `src/app/_data/cmsConfigSchema.json` is a file autogenerated when the server starts, with `predev` and `prebuild` commands. It does not require manual editing and should probably be part of `.gitignore` in the future.
```

Comment:
I'd love to git ignore `cmsConfigSchema.json` as part of this PR. Should we do it?

Reply:
We can't ignore `cmsConfigSchema.json` - It's used in our code.

Reply:
As per our discussion with Mirha, it can be ignored in a different PR
https://www.notion.so/filecoin/Ignore-autogenerated-files-6b211d6f97f64850bbd14f1c81d9114b?pvs=4